### DISCUSSION TITLE
point_0691

### DISCUSSION THREAD

**Discussion by andrea29-star** [cloudsen12/dataset#228] [Roy]
"20190131T073139_20190131T073619_T38RPN"
![image](https://user-images.githubusercontent.com/54947195/147497499-9039288c-df1a-4ba4-ae57-0a3746a84d31.png)

"20190421T072619_20190421T073927_T38RPN"
![image](https://user-images.githubusercontent.com/54947195/147497827-fa681159-9f8d-4284-bac0-47e5ab0aed29.png)
![image](https://user-images.githubusercontent.com/54947195/147497887-7b5552ab-a41e-430f-8620-16b5e8d37d63.png)

"20190401T072619_20190401T073229_T38RPN"
![image](https://user-images.githubusercontent.com/54947195/147498326-19209bdd-8d7f-4ec5-8529-d247274fd7e3.png)
A mi parecer en esta imagen debería tener 4 de dificultad

"20191217T073219_20191217T073312_T38RPN"
![image](https://user-images.githubusercontent.com/54947195/147498601-b062699d-5244-4b06-a827-d4aabd02fda5.png)

@ryali93

**Comment by AntoniCesar** [comment]
"20190131T073139_20190131T073619_T38RPN"
Se  le considero como nube gruesa debido a que existe una contraste con la nube delgada y debido a  la jerarquia se opto por definirlo como "nube gruesa". 
![Sin título](https://user-images.githubusercontent.com/86502862/152624274-5e787803-0b5e-4094-8917-920ae6feeea1.png)
